### ISSUE TITLE
Feature Request: Make it as plugin for prettier

### ISSUE THREAD

**Issue by iki** [inferrinizzard/prettier-sql#74]
**Describe the Feature**

- Create a prettier plug-in [`@prettier/plugin-sql`](https://prettier.io/docs/en/plugins.html)
- Optional: remove standalone VSCode plugin - it's simpler to avoid maintenance and focus on functionality within Prettier

**Why do you want this feature?**

Firstly, thank you for your great work! This formatter has much more usable settings than [prettier-plugin-pg](https://github.com/benjie/prettier-plugin-pg) with [sql-formatter](https://github.com/zeroturnaround/sql-formatter) that misses even basic settings like line width.

However, there're some good features of [prettier](https://github.com/prettier/prettier):
- lint/format all files in project
- project based config and ignore files (also respected by [VSCode prettier plug-in](https://github.com/prettier/prettier-vscode)) that can be committed to repository and shared across team, this was already requested in #61 

Rather than implement and maintain that, it's imo simpler to publish a prettier plug-in and have benefits not only of the functionality, but also larger adoption in prettier [community](https://prettier.io/docs/en/plugins.html#community-plugins)

**Info**

- How would you be calling / using the script? 
  - `prettier` from command line
  - existing lint/format scripts in `package.json` using `prettier`
  - [VSCode prettier plug-in](https://github.com/prettier/prettier-vscode)
- What SQL language(s) does this apply to?  `any`
- What Node version? (If applicable) `n/a`

**Comment by inferrinizzard** [comment]
Yes, I plan on eventually releasing a prettier plugin as a peer release - along with the main npm package.
Unfortunately, I will have my hands full working on the 5.2.x and 6.x releases once I return from hiatus so I would appreciate if you or someone you know would be interested in working on this :)
